src/app/components/video-course/video-course.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component } from '@angular/core'
+import { MatCardModule } from '@angular/material/card'
+
+@Component({
+  selector: 'app-video-course',
+  standalone: true,
+  imports: [MatCardModule],
```

Comment:
In order to avoid a big list of material modules, you can import your MaterialModule here as well